user-service/README.MD

```diff
@@ -0,0 +1,19 @@
+# PeerPrep Backend for User Service - Developer Readme 
+
+## Quick Start
+
+- This backend service uses <b>npm</b> as the package manager
+- Why not yarn? Because i don't like.
+
+### How to run backend server
+
+1. Add the `USER_SERVICE_MONGODB_URL` (which max hasn't sent in tele chat) into the `.env` file
+2. Run `npm run dev` (make sure you are in the `backend/user-service/` directory)
+    - If it works, you should see the following console logs in your terminal
+  ```
+        Server is running on port 3000
```

Comment:
i think u forgot to change to 4000 but its okay can edit in another pr

Reply:
Ok fixed.